graphics/twm4nx/src/ctwm4nx.cxx

```diff
@@ -382,7 +382,10 @@ void CTwm4Nx::genMqName(void)
   unsigned long randvalue =
     (unsigned long)std::random() & 0x00fffffful;
 
-  std::asprintf(&m_queueName, "Twm4Nx%06lu", randvalue);
+  if (std::asprintf(&m_queueName, "Twm4Nx%06lu", randvalue) < 0)
+    {
+      m_queueName = NULL;
```

Comment:
why this is needed?

Reply:
Sets `m_queueName` to a known value in case `asprintf` fails. See explanation above.